docs/pim/symbol_attribute_type.md

```diff
@@ -0,0 +1,138 @@
+---
+description: Create a symbol attribute type that enables for the efficient representation of string-based values while enforcing their format in product specifications.
+---
+
+# Symbol attribute type
+
+In product specifications, the symbol attribute type enables the efficient representation of string-based data and enforces their format.
+
+This feature allows you to store standard product identifiers (such as EAN or ISBN) in the [Product Information Management](pim_guide.md) system.
+
+## Installation
+
+### Download the bundle
+
+To get the most recent stable version of this bundle, open a command terminal, navigate to your project directory, and run the following command:
+
+``` bash
+composer require ibexa/product-catalog-symbol-attribute
+```
+
+### Enable the bundle
+
+Symfony Flex enables and configures the `IbexaProductCatalogSymbolAttributeBundle` automatically.
+If you don't use it, you can manually enable this bundle by adding the line below to the Kernel of your project:
+
+``` php
+// config/bundles.php
+
+return [
+   // ...
+   Ibexa\Bundle\ProductCatalogSymbolAttribute\IbexaProductCatalogSymbolAttributeBundle::class => ['all' => true],
+   // ...
+];
+```
+
+### Update database schema
+
+To store symbol attribute values, the `IbexaProductCatalogSymbolAttributeBundle` needs an extra table.
+The following SQL query can be used to build the required database structure:
+
+``` sql
+create table ibexa_product_specification_attribute_symbol (
```

Comment:
The comment above is still valid, could you please reach out to someone from the backend team to provide the sql for postgresql?